git-sha.json

```diff
@@ -0,0 +1 @@
p5icqlrqq67bn5srxahcl7wjbk2hzldx59ifx6dg4g6", "simple-json": "1ygjvz95cyxgzyp5s0i37bqvmivfl3d97ra9ih81v4m08pypffy7", "simple-json-generics": "1izbrh9614yi0lzpnqbn9q7hbllhvvhrgyziganj7rzgphwn3ywx", "smolder": "0qgzf1dlmhqn4c1dcj6rk1hgzda0xls3lknmrcyvvvdlc39snqk1", "spec": "1kckpsjl9vgpqp86aw5gmf4as0ff6vx3x9zl60sxwiqc6pazxx3q", "spec-discovery": "1ddjwwmiw1h9kjs48mqb99xhx1s43s1d4yijxvpkccnirjrvlmar", "spec-quickcheck": "1h41djivqqc7lzn038y18j9kbdi1gahbl79gy9iilcyh8b6axq0d", "st": "0m2jkb9dmpbr8s1c20l7sm2q11y5rx8gqfiyspnyhq5apzkknjr0", "string-parsers": "1lykswyd3icv9lx4r096lxfd244i0pzj2f4fpm604czzlfw6bspv", "strings": "147l3l3fzn7liparhm2y3p8j4ck1lblra5j493p2hy5yy5ndznc8", "strongcheck": "0c30w9y6mabphjiizk69hv43ahb1x8rg960nz4mccxb8hnj26z9x", "sunde": "1s62al6ark10hqc6kdz3nqarblpa6fmdy5kb3bjlxsa8nib436pa", "svg-parser-halogen": "134jqv2ridx61s6czss7i1fgwfb0cw2pccavdzlcphxqhzng4n8v", "tailrec": "0z7k80nl8dgv8mc2w8xsl2n0637bd1l8ppxak8kaifgjjwa81hx3", "test-unit": "0iyqdrhb0n5qpck8cprl68034ivywmphbw5wr7zy1jlyj9450nwh", "these": "0ywwpbcz1d0pdi3f9h9kla52vq1if8zwdz7jq7lqz5s8zj8kyg5r", "toppokki": "00blhsk7j3xhvp16kj2qlmzd01g1v8nn154gr21l4krxrdb5553y", "tortellini": "0gpgr4f2l5rn1zy0iqy42gpll01abb11rd7s9gx9yzcj43l1fsgb", "transformers": "1aazy1zk66lng8w0gjx2l7sqfr968gmibdxi4kd93zb7bw5vldvn", "tuples": "045nsy0r2g51gih0wjhcvhl6gfr8947mlrqwg644ygz72rjm8wq4", "type-equality": "1b7szyca5s96gaawvgwrw7fa8r7gqsdff7xhz3vvngnylv2scl3w", "type-isequal": "0bsz9d160kpih35xdbd617x48c394spdmk1dj9v6mhsii87hwa54", "typelevel": "1gmkazrb68cj41vjldvqds0sxdvdxmn5l238bsgz89df82yamc1c", "typelevel-prelude": "0dn95n9jnk2ilw38cf9p8p6q3xad6ck1rq3r4jba92kmk4ql665r", "unfoldable": "077vl30j3pxr3zw6cw7wd0vi22j92j8va15r26rn53wzbzcgr41j", "unicode": "1a53jv7pzyjk5v6kmwwy50d3l8d26k0id59sn8g3lzkih24nalhp", "unordered-collections": "09rrsym4w0xainsxzgd1ma6ay4iyx2kbb3x3w4k2bp552bq3vmj1", "unsafe-coerce": "0k9255mk2mz6xjb11pwkgfcblmmyvr86ig5kr92jwy95xim09zip", "unsafe-reference": "0q758dz59qz0li4s3w1qcg921xp5i5rh6i1l611iv7rr8cbj11al", "validation": "1y5i1jpr6jn1yjjpi4l5mk7jyd5v21fym892s0y5bdn4nj7i465l", "variant": "1awxcl3kc3z3hm8kxdg0w74v76pj636nk31d4ggqh49y25v3mpxj", "web-clipboard": "0zbhrm4ck8a8wf1knpmxccfyws07frld8i70wnkwjfan1pgdji63", "web-dom": "070ybc5xzpsh75p618ll9j0lzxqgvc4cl5cdrn71v0vagmyhgm8h", "web-events": "0hymk08prp29dm8v3xq079m7inkspb6gsyyynwqrcxsl3mha1f51", "web-file": "14kzwrwbfacxr4krfriy8qx13wnhiydrgs6hp59a7bmx58w0ifks", "web-html": "1y0g2ginqmfng058c99dm9n25vhz3g76d4l2v7yyrln9w04m4mvx", "web-socket": "1dld6dvwyhpxwcj28grnndjbamdfmy2imn7sl2824n74s03sy9kj", "web-storage": "1lr4lswsp62kl21rl6jb377a0sya105sfimkxm3y22grx88l6vj1", "web-touchevents": "1avg7jljd1j7d0b7k7fbqxi90aa01awp9x17v53mynzg6gyvc8zb", "web-uievents": "1683ysabddnhsjiy70axpslagi2cp2ab3qd0r1qjx6ab2zjd6kc0", "web-xhr": "05s3wp525wyvfz3rv456shvng9c0a4ncjdc7yxn2zglhq973km23", "xiaomian": "1n2ghz1z04xqxmjqdh0phnslq6fc7027j70si0lbp7njic6xljjp", "yargs": "10xbiyghplg54n9p7vs7mba9gmqfsbyiz0045c5p4p3v6djj60w3" }
```

Comment:
Does this change often? Diffs are going to be hard to spot on one line like this, so if it's generated is there a way to have pretty printed?

Reply:
The file shouldn’t be checked in in the final version. Just to make it easy to try it out for now.

Reply:
I don’t know whether I’d like to keep the perl script to be honest, we already have some Haskell setup.